test/test_geometries/custom_lon_lat_point.hpp

```diff
@@ -72,14 +72,7 @@ struct access<rw_lon_lat_point<CoordinateType, CoordinateSystem>, Dimension>
     void set(rw_lon_lat_point<CoordinateType, CoordinateSystem>& p,
              CoordinateType const& value)
     {
-        if (Dimension == 0)
-        {
-            p.longitude = value;
-        }
-        else
-        {
-            p.latitude = value;
-        }
+        ( Dimension == 0 ? p.longitude : p.latitude ) = value;
```

Comment:
Thanks for the link.
OK, if this is standard, and supported by all compilers we support, we are allowed to use it. It is indeed much shorter.
So I changed opinion, no need to change, unless we later see in the matrix that this breaks some compilers which are for the rest working for us.